src/schemas/clinical-resolvers/commitClinicalSubmission.ts

```diff
@@ -23,24 +23,22 @@ import { convertClinicalSubmissionDataToGql } from '../utils';
 import { DeepReadonly } from 'deep-freeze';
 import { ActiveClinicalSubmission } from '../../submission/submission-entities';
 
-const commitClinicalSubmissionResolver = {
-  commitClinicalSubmission: async (
-    obj: unknown,
-    args: { programShortName: string; version: string },
-    contextValue: any,
-  ) => {
-    const { programShortName, version } = args;
-    const submissionData = <DeepReadonly<ActiveClinicalSubmission>>(
-      await submissionAPI.commitActiveSubmissionData(
-        programShortName,
-        version,
-        (<GlobalGqlContext>contextValue).egoToken,
-      )
-    );
-    return convertClinicalSubmissionDataToGql(programShortName, {
-      submission: submissionData,
-    });
-  },
+const commitClinicalSubmission = async (
+  obj: unknown,
+  args: { programShortName: string; version: string },
+  contextValue: any,
+) => {
+  const { programShortName, version } = args;
+  const submissionData = <DeepReadonly<ActiveClinicalSubmission>>(
```

Comment:
is there one single way we can work around all this DeapReadonly stuff and stick with that approach?
I think this works out the same as using `as DeepReadonly....`
A utility function or a type guard that can be the de facto way for reusability, single point of functionality?

Reply:
I think this one we can handle by casting the return type of `commitActiveSubmissionData`: `{} as DeepReadonly<ActiveClinicalSubmission>`

https://github.com/icgc-argo/argo-clinical/commit/7b08b5677bb7d588f3f43fe4dd133751909811f8